src/libsync/networkjobs.cpp

```diff
@@ -568,7 +568,6 @@ CheckRedirectCostFreeUrlJob::CheckRedirectCostFreeUrlJob(const AccountPtr &accou
 
 void CheckRedirectCostFreeUrlJob::start()
 {
-    setFollowRedirects(false);
```

Comment:
I would say that this change is wrong
from a guess on the class name, it seems we wanted to check if a redirect would happen or not
so having Qt follow redirects automatically may be wrong here
that said, I guess I am missing the context of the `CheckRedirectCostFreeUrlJob` class
if that was supposed to be able to detect network login portal then we should ensure we do not break this